public/admin/config.yml

```diff
@@ -644,3 +644,24 @@ collections:
           - label: Metadata description
             name: metadataDescription
             widget: text
+  - label: Vaccinations directory
+    name: vaccines
+    folder: 'public/cms-content/vaccines'
+    create: true
+    label_singular: Vaccine
+    description: description
+    format: json
+    identifier_field: state
+    fields:
+      - label: State
+        name: state
+        widget: select
+        options:
+          - { label: "Massachussetts", value: "MA" }
```

Comment:
We need to add all the options here, or transform it into a text field specify that is the state code.